### PR TITLE
Authentication page

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
Added Authentication 

1. Login page 
2. Sign up page 
3. Log out page 

snap shots

image

image

image

### CODE REVIEW

**Review by eakansh125 — APPROVED** [review]
Could not find any syntactic issue, just some pixel to rem that can be taken care of.
Looks good to me!

File: components/authentication-page/logout-page.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            <h1>Log out Succesful !</h1>
+            <div class="login-bottom">
+                <a href="/components/authentication-page/login-page.html">   <button>Log In</button>  </a>
+                
+                <a href="/index.html"> <button>Home</button></a>
+        </div>
+        
+    </main>
+
+
```
Comment (gauravsinhaweb) [inline components/authentication-page/logout-page.html]: remove unnecessary spacing.

File: components/authentication-page/signup-page.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            <label for="text-input" auto-complete="new-password">Password</label>
+            <input type="password" class="text-input" placeholder="Your password"><span class="login-icon"><i class="fas fa-eye"></i></span>
+        </div>
+
+        <div class="login-bottom"><button>Login</button>
+            <p>Already have an Account <a href="/components/authentication-page/login-page.html">Login in </a></p>
+        </div>
+    </main>
+
+
```
Comment (gauravsinhaweb) [inline components/authentication-page/signup-page.html:65]: remove unnecessary spacing.

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge after changes.

**Pull request merged**